pom.xml

```diff
@@ -23,7 +23,7 @@
 
     <groupId>bio.overture</groupId>
     <artifactId>rollcall</artifactId>
-    <version>2.1.1-SNAPSHOT</version>
+    <version>2.2.0</version>
```

Comment:
looks like previous release wasn't merged back, this PR needs to be merged after: https://github.com/overture-stack/rollcall/pull/37